src/main/java/uk/gov/pay/ledger/transaction/resource/TransactionResource.java

```diff
@@ -41,10 +41,15 @@ public TransactionResource(TransactionService transactionService) {
     @GET
     @Timed
     public TransactionView getById(@PathParam("transactionExternalId") String transactionExternalId,
-                                   @QueryParam("account_id") @NotEmpty String gatewayAccountId,
+                                   @QueryParam("account_id") String gatewayAccountId,
```

Comment:
Can we not use Drop wizard inbuilt validation and object for multiple params? https://github.com/alphagov/pay-ledger/blob/ab5d499f5943b13f0e5e398b0397fc946475b05e/src/main/java/uk/gov/pay/ledger/transaction/resource/TransactionResource.java#L55

We could use same object to build SQL too as here https://github.com/alphagov/pay-ledger/blob/c5b1c9c842a5324a4f3ddbddf863f06118f4be45/src/main/java/uk/gov/pay/ledger/transaction/search/common/TransactionSearchParams.java#L131